runtime/doc/msgpack_rpc.txt

```diff
@@ -116,6 +116,12 @@ functions can be called interactively:
     >>> nvim = attach('socket', path='[address]')
     >>> nvim.command('echo "hello world!"')
 <
+One can also spawn and connect to an embedded nvim instance via |rpcstart()|
+>
+    let vim = rpcstart('build/bin/nvim', ['--embed'])
```

Comment:
We should probably not assume a development environment for examples, so maybe `/usr/bin/nvim` would be better.